src/converse-chatview.js

```diff
@@ -763,18 +763,26 @@ converse.plugins.add('converse-chatview', {
             },
 
             parseMessageForCommands (text) {
-                const match = text.replace(/^\s*/, "").match(/^\/(.*)\s*$/);
-                if (match) {
-                    if (match[1] === "clear") {
+                text = text.replace(/^\s*/, "");
+                const command = (text.match(/^\/([a-zA-Z]*) ?/) || ['']).pop().toLowerCase();
+                if (command) {
+                    const args = text.slice(('/'+command).length+1).trim();
+                    if (command === "clear") {
                         this.clearMessages();
                         return true;
+                    } else if (command === "inject") {
```

Comment:
Here you need to check if debugging is turned on.

Reply:
I'd rather not check it here. Otherwise you might end up accidently entering raw xmpp-stanzas in a chat because you forgot to turn on debugging.

Reply:
Makes sense, then an error message needs to be shown informing the user that debug mode is not on.

Reply:
Done.